README.md

```diff
@@ -0,0 +1,113 @@
+# CreateProcessW
+
+This crate provide an API similar to `std::process` to create and handle process
+using the [windows-rs][windows-rs] crate and the Windows API (see [this example][creating-processes]).
```

Comment:
```suggestion
This crate provide an API similar to `std::process` to create and handle processes
on Windows using the Win32 API through the [windows-rs][windows-rs] crate
(see [this example][creating-processes]).

It's main difference with `std::process::Command` is that it allows running a command
string instead of having to pass the command executable and the arguments separately.

This is the equivalent of running:

std::process::Command("cmd.exe")
    .arg("/c")
    .arg(any_command_string)
    .spawn()

The only difference will be that the `Child` instance will use the PID of the command instead of
the PID of `cmd.exe`. This is important because calling `.terminate()` in the code above does
not work as it kills the PID of `cmd.exe` instead of the actual command that has been ran.
```